packages/noya-state/src/reducers/interactionReducer.ts

```diff
@@ -77,6 +83,11 @@ export type InteractionState =
     }
   | {
       type: 'editPath';
+      current?: Point;
```

Comment:
Let's call this `point` instead of `current` for both `'editPath'` and `'drawingShapePath'`, since it'll be clearer when used as `state.interactionState.point ? a : b`.